src/web/app/components/question-types/question-edit-answer-form/rank-options-question-edit-answer-form.component.ts

```diff
@@ -26,6 +26,9 @@ export class RankOptionsQuestionEditAnswerFormComponent
   }
 
   ngOnInit(): void {
+  }
+
+  ngOnChanges(): void {
```

Comment:
The population of answers should be in `ngOnInit` (I hope you know why and why it works)? I just realise that if you put the `triggerModelChange` here, it will result in infinite loop.